app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<p style="color: green"><%= notice %></p>
+<%= render partial: "recipe", locals: {recipe_display: 'details', recipe: @recipe} %>
+<div>
+  <%= link_to "Edit this recipe", edit_recipe_path(@recipe) %> |
+  <%= link_to "Back to recipes", recipes_path %>
+  <button data-micromodal-trigger="modal-1" type="button">Generate</button>
```

Comment:
- [ ] Good job so far Team 👏 👏 You've done a lot of work up to now Kudos!! In addition to what you've done, there is only one feature that is remaining to be implemented which is also important and it is the `Generate the shopping list` As you can see, the above button is supposed to show a popup to choose the inventory when clicked but currently there is nothing being called when someone clicks it.
As you can see from the screenshot below 👇 
![image](https://user-images.githubusercontent.com/90222110/233805680-beb906b2-eda8-4ffa-bf06-68fc8f8d4f41.png)

![image](https://user-images.githubusercontent.com/90222110/233805706-9ca555d1-d882-4812-84b1-419f5f94779f.png)

When someone clicks on the `Generate` button, a popup should appear to let the user choose the inventory to generate a shopping list from.
Then the app will compare the food listed as ingredients in the recipe to the food in the inventory and then make a list of missing ones and display them on the shopping list 💡 

That is basically what I would suggest that you implement otherwise good job for what you have done so far 😉